integration/flink/app/src/test/java/io/openlineage/flink/FlinkContainerUtils.java

```diff
@@ -37,7 +37,7 @@ public class FlinkContainerUtils {
   static MockServerContainer makeMockServerContainer(Network network) {
     return new MockServerContainer(
             DockerImageName.parse("jamesdbloom/mockserver:mockserver-5.12.0"))
-        .withLogConsumer(of -> consumeOutput("mockserver", of))
+        // .withLogConsumer(of -> consumeOutput("mockserver", of)) no need to consume its output
```

Comment:
Initially I thought it may be useful for someone in future to turn it on locally. Now I think you're right - I can't see scenario when this becomes useful